### PR TITLE
SPUI-493/496 > Add UI for Sign In

### PR DESCRIPTION
**Pull request by VovaMedvediev**

### CODE REVIEW

File: plugins/simple_kit/lib/src/agreements/simple_privacy_policy/simple_policy_text.dart [hunk 1]
```diff
@@ -24,21 +23,19 @@ class SPolicyText extends StatelessWidget {
...
-            onUserAgreementTap: onUserAgreementTap,
-            betweenText: betweenText,
-            privacyPolicyText: privacyPolicyText,
-            onPrivacyPolicyTap: onPrivacyPolicyTap,
-          ),
-        ],
-      ),
+    return Column(
+      children: [
+        const SpaceH34(),
```
Comment (eli1stark) [inline plugins/simple_kit/lib/src/agreements/simple_privacy_policy/simple_policy_text.dart:28]: No changes to simple_kit outside of ui_guides

**Review by eli1stark — CHANGES_REQUESTED** [review]

File: lib/auth/screens/login/login.dart [hunk 2]
```diff
@@ -28,86 +18,141 @@ class Login extends HookWidget {
...
+                            userAgreementText: 'User Agreement',
+                            betweenText: ' and the ',
+                            privacyPolicyText: 'Privacy Policy',
+                            onUserAgreementTap: () {},
+                            onPrivacyPolicyTap: () {},
+                          ),
+                        ),
+                      ),
+                      Material(
+                        color: Colors.grey[200],
```
[the remark this answers is not in the document]
Comment (VovaMedvediev) [reply lib/auth/screens/login/login.dart:109]: done

File: lib/auth/screens/login/login.dart [hunk 3]
```diff
@@ -28,86 +18,141 @@ class Login extends HookWidget {
...
+                          labelText: 'Password',
+                          onErrorIconTap: () {
+                            _showErrorNotification(notificationQueueN);
+                          },
+                          errorNotifier: passwordError.value,
+                        ),
+                      ),
+                      const Spacer(),
+                      Container(
+                        color: Colors.grey[200],
```
[the remark this answers is not in the document]
Comment (VovaMedvediev) [reply lib/auth/screens/login/login.dart:94]: done

File: lib/auth/screens/login/login.dart [hunk 4]
```diff
@@ -28,86 +18,141 @@ class Login extends HookWidget {
...
+          child: Expanded(
+            child: CustomScrollView(
+              slivers: [
+                SliverFillRemaining(
+                  hasScrollBody: false,
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      Padding(
+                        padding: EdgeInsets.symmetric(horizontal: 24.w),
```
[the remark this answers is not in the document]
Comment (VovaMedvediev) [reply lib/auth/screens/login/login.dart:61]: done

File: lib/auth/screens/login/login.dart [hunk 5]
```diff
@@ -28,86 +18,141 @@ class Login extends HookWidget {
...
+                          },
+                          onErrorIconTap: () {
+                            _showErrorNotification(notificationQueueN);
+                          },
+                          errorNotifier: emailError.value,
+                        ),
+                      ),
+                      const SDivider(),
+                      Padding(
+                        padding: EdgeInsets.symmetric(horizontal: 24.w),
```
[the remark this answers is not in the document]
Comment (VovaMedvediev) [reply lib/auth/screens/login/login.dart:79]: done

File: lib/auth/screens/login/login.dart [hunk 6]
```diff
@@ -28,86 +18,141 @@ class Login extends HookWidget {
...
+                            _showErrorNotification(notificationQueueN);
+                          },
+                          errorNotifier: passwordError.value,
+                        ),
+                      ),
+                      const Spacer(),
+                      Container(
+                        color: Colors.grey[200],
+                        child: Padding(
+                          padding: EdgeInsets.symmetric(horizontal: 24.w),
```
[the remark this answers is not in the document]
Comment (VovaMedvediev) [reply lib/auth/screens/login/login.dart:96]: done

File: lib/auth/screens/login/login.dart [hunk 7]
```diff
@@ -28,86 +18,141 @@ class Login extends HookWidget {
...
+                                );
+                              }
+                            },
+                          ),
+                        ),
+                      ),
+                      Container(
+                        width: double.infinity,
+                        height: 24.h,
+                        color: Colors.grey[200],
```
[the remark this answers is not in the document]
Comment (VovaMedvediev) [reply lib/auth/screens/login/login.dart:130]: done

File: lib/auth/screens/login/login.dart [hunk 8]
```diff
@@ -28,86 +18,141 @@ class Login extends HookWidget {
...
           loading: () {},
         );
       },
-      child: PageFrame(
-        header: 'Sign in to simple',
-        onBackButton: () => Navigator.pop(context),
-        resizeToAvoidBottomInset: false,
+      child: SPageFrame(
+        header: Padding(
+          padding: EdgeInsets.symmetric(horizontal: 24.w),
```
[the remark this answers is not in the document]
Comment (VovaMedvediev) [reply lib/auth/screens/login/login.dart:42]: done

File: lib/auth/screens/login/login.dart [hunk 9]
```diff
@@ -28,86 +18,141 @@ class Login extends HookWidget {
...
+                            privacyPolicyText: 'Privacy Policy',
+                            onUserAgreementTap: () {},
+                            onPrivacyPolicyTap: () {},
+                          ),
+                        ),
+                      ),
+                      Material(
+                        color: Colors.grey[200],
+                        child: Padding(
+                          padding: EdgeInsets.symmetric(horizontal: 24.w),
```
[the remark this answers is not in the document]
Comment (VovaMedvediev) [reply lib/auth/screens/login/login.dart:111]: done

File: lib/auth/screens/login/login.dart [hunk 10]
```diff
@@ -28,86 +18,141 @@ class Login extends HookWidget {
...
-      child: PageFrame(
-        header: 'Sign in to simple',
-        onBackButton: () => Navigator.pop(context),
-        resizeToAvoidBottomInset: false,
+      child: SPageFrame(
+        header: Padding(
+          padding: EdgeInsets.symmetric(horizontal: 24.w),
+          child: SBigHeader(
+            title: 'Sign in',
+            onBackButtonTap: () => Navigator.of(context).pop(),
```
[the remark this answers is not in the document]
Comment (VovaMedvediev) [reply lib/auth/screens/login/login.dart:45]: done

File: plugins/simple_kit/lib/src/agreements/simple_privacy_policy/simple_policy_text.dart (hunk 1 above)
Comment (VovaMedvediev) [reply plugins/simple_kit/lib/src/agreements/simple_privacy_policy/simple_policy_text.dart:28]: done

**Pull request merged**